backend/src/Controller/TrackController.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Track;
+use App\Repository\TrackRepository;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Serializer\Serializer;
+use Symfony\Component\Serializer\SerializerInterface;
+use Symfony\Component\Validator\Constraints\Json;
+
+class TrackController extends AbstractController
+{
+    /**
+     * @Route("/track", methods={"GET"})
+     */
+    public function index(SerializerInterface $serializer): JsonResponse
+    {
```

Comment:
I think we problems with the SerializerInterface, maybe this one crashes, if you have relations in your entities. Write your own serializer in this case

Reply:
Good point, I will keep that in mind!